testing/integration/agent_long_running_leak_test.go

```diff
@@ -68,7 +68,7 @@ func TestLongRunningAgentForLeaks(t *testing.T) {
 		Sudo:  true,  // requires Agent installation
 		OS: []define.OS{
 			{Type: define.Linux},
-			{Type: define.Windows},
+			// {Type: define.Windows}, // flaky for windows. Even running as privileged, it returns DEGRADED state.
```

Comment:
This change seems unrelated to this PR and should be in its own PR.

This needs to be be filed as a flaky test or if already filed linked here for reason for disablement.

I do not like to see windows being disabled, because its rare that others will pick it up and fix it.

Reply:
Thanks, I'll do that!